static/app/utils/dashboards/issueFieldRenderers.tsx

```diff
@@ -174,6 +182,24 @@ const issuesCountRenderer = (
   );
 };
 
+const getDiscoverUrl = (
+  data: EventData,
+  organization: Organization,
+  filtered?: boolean
+) => {
+  const commonQuery = {projects: [Number(data.projectId)]};
+  const discoverView = EventView.fromSavedQuery({
+    ...commonQuery,
+    id: undefined,
+    name: data.title,
+    fields: ['title', 'release', 'environment', 'user', 'timestamp'],
+    orderby: '-timestamp',
+    query: `issue.id:${data.id}${filtered ? data.discoverSearchQuery : ''}`,
+    version: 2,
```

Comment:
I'm seeing a discrepancy here, if my dashboard has a date range (eg. 24h) when i click this link that date range isn't carried over (eg. discover is at 14d) which means the counts don't match.